styles/search_dropdown.module.css

```diff
@@ -86,3 +86,36 @@
   background-color: #f2f5fd;
   outline: none;
 }
+
+/* Search bar media queries */
+
+@media (max-width: 450px) {
+  .searchBar {
+    font-size: 14px;
+    min-width: 250px;
+    width: 350px;
+  }
+}
+
+@media (max-width: 390px) {
+  .searchBar {
+    font-size: 12px;
+    min-width: 250px;
+    width: 300px;
+  }
+}
+
+@media (max-width: 335px) {
+  .searchBar {
+    font-size: 11px;
+    width: 275px;
+  }
+}
+
+@media (max-width: 300px) {
+  .searchBar {
+    font-size: 10px;
+    width: 250px;
+  }
+}
+
```

Comment:
It works but IMO it would be also recommended that if we can use flex property here instead media queries. 
Can use : 
* flex-wrap: wrap    // to wrap  bar and btn if screen size small
* flex-direction: row;  // to keep search bar and button inline

Also, few suggestions : 
* it seems search bar font size is too small for small screen, can we use standard font-size:1rem; it will take care of responsiveness.
* I think searchbar placeholder can be restructure to 'Enter pincode or district name'. to avoid overflowing content from search bar. 
* Search Button height should be equal to ~50px. 
* Button width: can be stretch to 160px max , keeping right and left padding equal.

Reply:
Adding screenshot where it is breaking : 
![image](https://user-images.githubusercontent.com/3944566/119240724-43e86600-bb6f-11eb-810b-3aee46c2eea1.png)